src/pages/Admin/Charity/EditProfile/index.tsx

```diff
@@ -42,12 +41,11 @@ export default function EditProfile() {
   );
 }
 
-function FormWithContext(props: TProfile) {
-  const { txResource, id, owner } = useAdminContext<"charity">(ops);
-
+function FormWithContext(props: TProfile & { id: number }) {
+  //
   const init: EndowmentProfileUpdate = toProfileUpdate({
     type: "initial",
-    data: { ...props, id, owner },
+    data: { ...props, id: props.id, owner: "is not relevant anymore" },
```

Comment:
@SovereignAndrey, @jp-mariano , I assume there would be a refactor on edit profile endpoint once web2 authentication is in place

Reply:
@ap-justin Yeah, `owner` will fall away, to be replaced by auth checks if a given `User` is authorised/linked to an `Endowment`.